examples/tls-authentication/nginx/README.md

```diff
@@ -0,0 +1,56 @@
+# TLS termination
+
+This example demonstrates how to enable the TLS Authentication through the nginx Ingress controller.
+
+## Prerequisites
+
+You need a valid CA File, composed of a group of valid enabled CAs. This MUST be in PEM Format.
+Also the Ingress must terminate TLS, otherwise this makes no sense ;)
+
+## Deployment
+
+The following command instructs the controller to enable the TLS Authentication using
+the secret containing the valid CA chains.
```

Comment:
The following command instructs the controller to enalbe TLS authentication using the secret from the `ingress.kubernetes.io/auth-tls-secret` annotation on the Ingress. Clients must present this cert to the loadbalancer, or they will receive a HTTP 400 response.